UrbanAirship.Android.FCM.nuspec

```diff
@@ -11,9 +11,10 @@
 
       <dependencies>
          <group targetFramework="MonoAndroid4.1">
-            <dependency id="Xamarin.Firebase.Messaging" version="71.1740.0" />
-            <dependency id="Xamarin.GooglePlayServices.Base" version="71.1610.0" />
-            <dependency id="urbanairship.android.core" version="14.0.1"/>
+            <dependency id="Xamarin.Firebase.Messaging" version="120.3.0" />
+            <dependency id="Xamarin.GooglePlayServices.Base" version="117.1.0" />
+            <dependency id="Xamarin.Google.Dagger" version="2.27.0" />
```

Comment:
I had to add this to resolve a runtime crash during Firebase initialization. It's a transitive dependency, but wasn't getting pulled in for some reason.